src/Faker/Provider/en_PH/Address.php

```diff
@@ -392,9 +392,153 @@ class Address extends \Faker\Provider\en_US\Address
     );
 
     protected static $barangay = array(
-        'Poblacion'
-    );
 
+        // Baguio City Barangays
```

Comment:
could you add a source as a `@link` phpDoc, and put a few more entries per line (e.g. all the entries starting with A in one line)?